src/main/resources/ipaas.yaml

```diff
@@ -12,7 +12,7 @@ defaultBuildParameters:
     deploy'
 
 builds:
-- name: atlasmap-2.0.2-${version.fuse.prefix}
+- name: atlasmap-2.0.7-${version.fuse.prefix}
```

Comment:
Atlasmap is 2.0.6 for syndesis/1.11.x
https://github.com/jboss-fuse/syndesis/blob/1.11.x/app/pom.xml#L42
And there is no 2.0.7 release yet
https://github.com/atlasmap/atlasmap/releases